apps/penxle.com/src/routes/(default)/search/(index)/+page.svelte

```diff
@@ -99,13 +99,13 @@
 
             <a class="flex flex-col gap-1 grow" href={`/${space.slug}`}>
               <p class="body-15-b">{space.name}</p>
-              <p class="body-14-m text-secondary">{space.description ?? ''}</p>
+              <p class="body-14-m text-secondary line-clamp-2">{space.description ?? ''}</p>
             </a>
 
             {#if !space.meAsMember}
               {#if space.followed}
                 <button
-                  class="py-1.5 px-2 rounded-12 border border-secondary body-13-m flex items-center gap-1"
+                  class="py-1.5 px-2 rounded-12 border border-secondary body-13-m flex items-center gap-1 whitespace-nowrap"
```

Comment:
```suggestion
                  class="py-1.5 px-2 rounded-12 border border-secondary body-13-m flex items-center gap-1 shrink-0"
```

P3: flex item 인데 flex-shrink: 0 으로 하면 해결되지 않을까요? (구두로 바로 쓴 리뷰라 직접 확인 해보셔야 함)